rln-js/index.html

```diff
@@ -44,29 +44,33 @@ <h2 class="mu1">Credentials</h2>
 
     <div class="row">
       <div class="w50">
-        <h4>You can either generate new credentials:</h4>
-        <button disabled id="generate-credentials" type="button">
-          Generate RLN Credentials
-        </button>
+        <h4>Generate new, or import existing, credentials from wallet:</h4>
         <br />
+        <div id="import-from-wallet">
+          <button id="import-from-wallet-button" type="button">
+            Fetch credentials from wallet
```

Comment:
I'd keep it to "Generate RLN Credentials".

Reply:
May I suggest an "advanced" field as well, where the user can edit the message they sign to derive the seed for the keygen?

Would be cool!

Reply:
> May I suggest an "advanced" field as well, where the user can edit the message they sign to derive the seed for the keygen?
> 
> Would be cool!

Agreed; tracking [here](https://github.com/waku-org/js-waku-examples/issues/166) :)